easybuild/easyconfigs/b/Boost/Boost-1.74.0-library_version_type_serialization.patch

```diff
@@ -0,0 +1,11 @@
+diff -rub serialization-boost-1.74.0_orig/include/boost/serialization/set.hpp serialization-boost-1.74.0/include/boost/serialization/set.hpp
```

Comment:
@deniskristak Please add a comment on top of the patch file with a pointer to the upstream fix (and don't forget to update the SHA256sum in the easyconfig file accordingly)

Reply:
fixed in 39cecca